charts/streams-app/values.yaml

```diff
@@ -23,29 +23,29 @@ resources:
     memory: 2G
 
 streams:
-#  brokers: "test:9092"
-#  schemaRegistryUrl: "url:1234"
+  # brokers: "test:9092"
```

Comment:
Isn't the indentation wrong now? When uncommenting it, the values should be correct

Reply:
previously they were weirdly formatted with the hash position at the beginning of the line. compare to autoscaling config below which is formatted correctly